mlflow/sklearn/__init__.py

```diff
@@ -567,11 +567,13 @@ def autolog():
 
         - Common metrics for regressor:
 
-          - `(root) mean squared error`_
+          - `mean squared error`_
 
-          .. _(root) mean squared error:
+          .. _mean squared error:
               https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html
 
+          - `root mean squared error`
```

Comment:
Can you remove the backticks here to avoid italicizing `root mean squared error`?

Reply:
<img width="352" alt="Screen Shot 2020-09-03 at 0 03 57" src="https://user-images.githubusercontent.com/17039389/92000879-ff1a6600-ed78-11ea-98c0-5dca13649d3e.png">

Reply:
Corrected.